ngx-tools/src/return-array-value-from-hash/return-array-value-from-hash.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Helper function to return an array of values from an hash object
+ *
+ * @param {Array} keys The array containing the key values (number|string) to retrieve from the hash
```

Comment:
With the addition of TypeScript typings we no longer need to define the types in our JSDoc comments.

So our doc comments can be shortened to:

```
@param paramName - description
@return description
```

```
// example:
@param keys - The array...
@param hash - The object to
@return The array of...
```